lib/js/src/manager/screen/utils/_PresentAlertOperation.js

```diff
@@ -227,6 +232,19 @@ class _PresentAlertOperation extends _Task {
             } else {
                 console.log('All alert images uploaded');
             }
+
+            const artworkResults = [];
+            for (let index = 0; index < successes.length; index++) {
+                artworkResults.push({
+                    artwork: artworksToBeUploaded[index],
+                    success: successes[index],
+                });
+            }
+            // only keep artworks that have been uploaded successfully by the file manager
+            const artworkNamesSuccessfullyUploaded = artworkResults.filter(result => result.success).map(result => result.artwork.getName());
```

Comment:
It seems odd to me here that we have a loop to build `artworkResults` into an array of JSON objects only for it to be filtered and torn down immediately after. 
We should be able to condense this to one loop that simply checks the success result, then only pushes the artwork name to the array when `success=true`.
```suggestion
            const artworkNamesSuccessfullyUploaded = [];
            for (let index = 0; index < successes.length; index++) {
            // only keep artworks that have been uploaded successfully by the file manager
                if (successes[index]) {
                    artworkNamesSuccessfullyUploaded.push(artworksToBeUploaded[index].getName());
                }
            }
```